lib/main.js

```diff
@@ -100,6 +109,8 @@ export function activate () {
 export function minimapViewProvider (model) {
   if (model instanceof Minimap) {
     const element = new MinimapElement()
+    // add DOMStylesReaderInstance
+    element.DOMStylesReader = DOMStylesReaderInstance;
```

Comment:
The mixin hackery that has remained from the old times prevents me to pass this variable to the constructor like a normal human 😞